### DISCUSSION TITLE
Playable version of Game of Life for RP6502

### DISCUSSION THREAD

**Discussion by grakoczy** [picocomputer/community#128] [Q&A]
Hi,

Based on game of life example I've made playable version of the game.
It runs at about 5 iterations per second (depends on amount of living cells).
You can paint/erase your own cells. I've also added two example shapes you can place on the map.
**[https://github.com/grakoczy/picocomputer-game-of-life/tree/main](https://github.com/grakoczy/picocomputer-game-of-life)**

https://github.com/user-attachments/assets/de1190d1-2a1b-4a0a-aab7-223e5389aff3

**Comment by WojciechGw** [comment]
excelent :) thank You
Wojtek

**Comment by rumbledethumps** [comment]
Fast!

**Comment by WojciechGw** [comment]
Hello,

@grakoczy 
maybe You'll like my version of Your work ...
I made a few minor changes
please share Your impressions

[life.zip](https://github.com/user-attachments/files/17542390/life.zip)

source files

[source[game-of-life].zip](https://github.com/user-attachments/files/17542802/source.game-of-life.zip)

**Comment by grakoczy** [comment]
Hi,
I love your version! 
It's much better now. I've already merged those changes to the repository. 
Thank you!

--
Grzegorz

**Comment by rumbledethumps** [comment]
Amazing!
![Screenshot 2024-10-28 233151](https://github.com/user-attachments/assets/79458103-d7bc-4236-8218-d78e0322fa7e)

**Comment by WojciechGw** [comment]
Thank You :)

run out of memory to add some ideas 
like shape editor, sounds etc.
